examples/resources/boundary_worker/controller_led.tf

```diff
@@ -0,0 +1,5 @@
+resource "boundary_worker" "controller_led" {
+  scope_id    = "global"
+  name        = "worker 1"
```

Comment:
Should we give the workers different names to reinforce the idea that resources need to have unique names?

Reply:
Good catch. I’ll change it to “controller-led-worker-1” and “worker-led-worker-1”.​